lib/iris/cube.py

```diff
@@ -997,6 +1044,23 @@ def remove_coord(self, coord):
         for factory in self.aux_factories:
             factory.update(coord)
 
+    def remove_cell_measure(self, cell_measure):
+        """
+        Removes a cell measure from the cube.
+
+        Args:
+
+        * cell_measure (CellMeasure)
+            The CellMeasure to remove from the cube.
+
+        See also
+        :meth:`Cube.add_cell_measure()<iris.cube.Cube.add_cell_measure>`
+
+        """
+        self._cell_measures_and_dims = [(cell_measure_, dim) for cell_measure_,
+                                        dim in self._cell_measures_and_dims
+                                        if cell_measure_ is not cell_measure]
```

Comment:
@marqh Whoa! We need to be careful here ...

Your reconstructing the `Cube._cell_measures_and_dims` list to contain **tuples** i.e. `(cm, dims)`.

When you add a new cell measure with `Cube.add_cell_measure` it appends to `Cube._cell_measures_and_dims` with a **list** i.e. `[cm, dims]`

We need to be consistent in our approach, as I notice within `Cube.__getitem__` there's the call ...

```python
if self._cell_measures_and_dims != cube._cell_measures_and_dims:
```